src/test/java/io/opentracing/contrib/concurrent/TracedExecutorServiceTest.java

```diff
@@ -24,6 +24,10 @@ protected ExecutorService toTraced(ExecutorService executorService) {
     return new TracedExecutorService(executorService, mockTracer);
   }
 
+  protected ExecutorService toTracedNoParent(ExecutorService executorService) {
```

Comment:
`s/toTracedNoParent/toTracedCreatingParent/`? When reading the tests, it's a bit confusing that `toTracedNoParent` has more spans than their `toTraced` counterparts.

Reply:
I updated method

Reply:
"No parent required" still implies the same to me :-)

Reply:
renamed :)

Reply:
Thanks for your patience on this one. Looks great now.